### PR TITLE
Update Multi-Line Variable Docs

### PR DESCRIPTION
**Pull request by bensojona**
Document and discuss how to use both "Environment Variables" and "Terraform Variables" to store multi-line files in Atlas.

### CODE REVIEW

**Comment by bensojona** [comment]
https://app.intercom.io/a/apps/wvqhq7d7/inbox/jon@hashicorp.com/conversations/1506626940

File: terraform/runs/variables-and-configuration.html.md [hunk 1]
```diff
@@ -66,24 +66,52 @@ resource was created outside of GitHub (like using `terraform push`).
...
 
-Atlas has the ability store multi-line files as variables. The recommended way to manage your secret/sensitive multi-line files (SSL keys, certificates, etc.) is to add them as [Terraform Variables](#terraform-variables) in Atlas.
+Atlas has the ability to store multi-line files as variables. The recommended way to manage your secret/sensitive multi-line files (private key, SSL cert, SSL private key, CA, etc.) is to add them as [Terraform Variables](#terraform-variables) or [Environment Variables](#environment-variables) in Atlas.
 
-Just like secret strings, it is recommended that you never check-in these multi-line secret files to version control. Best practice is setting a blank [variable in Terraform](https://www.terraform.io/docs/configuration/variables.html) that resources utilizing the file will reference, `terraform push` to Atlas, navigate to the "Variables" section of your Atlas Environment, paste the contents of that file into the variable and save. See example of setting variables below.
+Just like secret strings, it is recommended that you never checkin these multi-line secret files to version control by following the below steps.
```
Comment (KFishner) [inline terraform/runs/variables-and-configuration.html.md]: "check in"

File: terraform/runs/variables-and-configuration.html.md [hunk 2]
```diff
@@ -66,24 +66,52 @@ resource was created outside of GitHub (like using `terraform push`).
...
+
+    $ terraform push -name $ATLAS_USERNAME/example -var "private_key=$MY_PRIVATE_KEY"
+
+`terraform push` any "Environment Variables" to Atlas:
+
+    $ TF_VAR_private_key=$MY_PRIVATE_KEY terraform push -name $ATLAS_USERNAME/example
+
+Alternatively, you can add or update variables manually by going to the "Variables" section of your Atlas Environment and pasting the contents of the file in as the value.
+
+Now, any resource that consumes that variable will have access to the file without having to check it in to version control. If you want to run Terraform locally, that file will still need to be passed in as a variable in the CLI. View the [Terraform Variable Documentation](https://www.terraform.io/docs/configuration/variables.html) for more info on how to accomplish this.
```
Comment (KFishner) [inline terraform/runs/variables-and-configuration.html.md]: "will have access to the variable value, without having to check the file into version control"

want to make it clear that you aren't using a file

**Comment by bensojona** [comment]
@KFishner forgot this guy was still open, good to merge?

**Comment by KFishner** [comment]
LGTM

**Pull request merged**